app/admin/page.rb

```diff
@@ -12,7 +12,7 @@ def find_resource
     f.inputs 'Details' do
       f.input :menu_title
       f.input :title
-      f.input :body
+      f.input :body, :as => :ckeditor
```

Comment:
Excellent. You might have a styling issue, where the editor is pushed away from the left side of the editing area. I fixed that using:

``` diff
- f.input :body, as: :ckeditor
+ f.input :body, as: :ckeditor, label: false
```